src/app/base/components/StatusBar/StatusBar.tsx

```diff
@@ -26,14 +26,44 @@ import type { UtcDatetime } from "@/app/store/types/model";
 import { NodeStatus } from "@/app/store/types/node";
 import { formatUtcDatetime, getTimeDistanceString } from "@/app/utils/time";
 
-const getLastCommissionedString = (machine: MachineDetails) => {
+const getLastCommissionedString = (machine: MachineDetails): string => {
```

Comment:
Bit of a nitpick, but I think this would be a lot more readable if placed *after* `findLastCommissioningTime` and `formatLastCommissioningTime`